gremlin-python/src/main/python/gremlin_python/driver/client.py

```diff
@@ -107,20 +109,12 @@ def _fill_pool(self):
             self._pool.put_nowait(conn)
 
     def close(self):
-        if self._sessionEnabled:
-            self._close_session()
+        logging.info("Closing Client with url '%s'", self._url)
         while not self._pool.empty():
             conn = self._pool.get(True)
             conn.close()
         self._executor.shutdown()
 
-    def _close_session(self):
```

Comment:
You removed this little function and I didn't see it's behavior anywhere else. That effectively undoes the work of #1510 - could you please make sure the "close" message gets sent when a session is closed just prior to closing the websocket connection?

Reply:
Will fix this, I think this was an accidental change that came in somehow from the __close_session being renamed from _close_session in the Transaction implementation.